contents/newsletter/first-time-founders.md

```diff
@@ -0,0 +1,133 @@
+
+3. **Building a solution that isn't 10x better or cheaper:** Points 1 and 2 inevitably lead here, as Taylor explains:
+
+> "We were very descriptive about the solution we were building and in the end it wasn't appreciably better than the email and spreadsheets based workflow they were using. People don't like change, and they won't adopt a new way of doing things unless it is massively better."
+
+## More first-time founder mistakes
+
+### Not solving a real problem
+
+Exhibit A: [Humane AI Pin](https://www.theverge.com/24126502/humane-ai-pin-review)
+
+### The wrong co-founders
+
+"[Companies are mostly about people](https://newsletter.posthog.com/p/what-we-learned-about-hiring-from)" and that starts with your co-founders. Marty Kausus quit his second startup the moment he found "the right co-founders" to start a more successful venture.
+
+They got together and "pivoted weekly for ~2 months" before landing on Pylon, a Zendesk/Intercom alternative for managing customer operations via Slack. None of his previous co-founders could commit to this level.
+
+> "My biggest takeaway on co-founders is to make sure they have similar hustle, spirit, and desired lifestyle. For Pylon, we wanted to live together and live and breathe the company. We joke that we don't have "work-life balance", we have "work-life integration" and that's ideal for us, while most people would find that miserable."
+
+### Not using analytics
+
+YC's Michael Seibel says that, even now, "we see this time and time again." See also: [What engineers get wrong about analytics](https://newsletter.posthog.com/p/what-engineers-get-wrong-about-analytics).
+
+### Just shipping what users say they want
+
+Talking to customers is important, but it can go too far. This was a lesson James Evans (CommandBar LINK) learned in his first startup:
+
+> "Our only real advantage relative to competitors was speed and eagerness, so we shipped a TON of features. Users would ask for something on Intercom in the morning and receive it that evening. Just crazy customer service."
+
+It felt good, but it "created a ton of product debt" and led them to ship "things that we didn't really want users to use... it made our product kind of Franksteiny-freakish." 
+
+With CommandBar, he was determined to build a more opinionated product. They only build new features for customers when they can say, with confidence "this feature will help your users and not annoy them."
+
+### Poor founder-market fit
+
+Per Borgen and his co-founder spent three years trying to make their first startup work. Despite reaching over a million downloads for their children's books for iPad idea, the venture almost broke his spirit: "I was convinced I'd never have the energy to do a startup again."
```

Comment:
be good to name company/whatever behind the iPad thing, and give it a date/when it launched so there's a sense of time between this failure and the subsequent success